test/IISIntegration.FunctionalTests/Inprocess/TestServerTest.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using IISIntegration.FunctionalTests.Utilities;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Testing.xunit;
+using Microsoft.Extensions.Logging.Testing;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace IISIntegration.FunctionalTests.Inprocess
+{
+    [SkipIfHostableWebCoreNotAvailible]
+    public class TestServerTest: LoggedTest
+    {
+        public TestServerTest(ITestOutputHelper output = null) : base(output)
+        {
+        }
+
+        [ConditionalFact]
+        public async Task Test()
+        {
+            var helloWorld = "Hello World";
+            using (var testServer = await TestServer.Create(ctx => ctx.Response.WriteAsync(helloWorld), LoggerFactory))
```

Comment:
Assert something in this test as that is one of the main reasons to make this change.